penlight-scm-4.rockspec

```diff
@@ -8,17 +9,20 @@ source = {
 
 description = {
   summary = "Lua utility libraries loosely based on the Python standard libraries",
-  homepage = "https://lunarmodules.github.io/Penlight",
+  detailed = [[
+Penlight is a set of pure Lua libraries focusing on input data handling
+(such as reading configuration files), functional programming
+(such as map, reduce, placeholder expressions,etc), and OS path management.
+Much of the functionality is inspired by the Python standard libraries.
+]],
   license = "MIT/X11",
+  homepage = "https://lunarmodules.github.io/Penlight",
+  issues_url = "https://github.com/lunarmodules/Penlight/issues",
   maintainer = "thijs@thijsschreijer.nl",
-  detailed = [[
-Penlight is a set of pure Lua libraries for making it easier to work with common tasks like
-iterating over directories, reading configuration files and the like. Provides functional operations
-on tables and sequences.
-]]
 }
 
 dependencies = {
+  "lua >= 5.1",
```

Comment:
this is tricky. Should we skip this until a next major release?

Reply:
This might not even be correct. How does the Moonscript VM identify itself?

If you want to save a rockspec rename until next release cycle I can split that out and stash it for now.

Reply:
I yanked this and will move it to another PR for later review, maybe right before a release.